Resources/views/sitemap/sitemap.xml.twig

```diff
@@ -3,7 +3,9 @@
 
 {#  {% include '@BaseTheme/sitemap/sitemap-element.xml.twig' with {'element': home} only %}#}
 
-{% for page in pages if page.translation.Available %}
+
+
+{% for page in pages if page.translation.Available and page.nodeTypename != 'NSLink' %}
```

Comment:
`if` in `for` are now deprecated in Twig 2.x.

https://twig.symfony.com/doc/2.x/deprecated.html

Reply:
Thx, I learn something.

I fix this with `filter`

```twig

{% for page in pages|filter(p => p.translation.Available and page.nodeType.name != 'NSLink') %}
   
```